test/actions/http_test.exs

```diff
@@ -416,8 +426,10 @@ defmodule WorkflowEngine.Actions.HTTPTest do
 
       assert log =~ "failed with status 400"
 
-      assert error =~ "bad_request"
-      assert error =~ "Something went wrong"
+      # INFO: is this supposed to be recoverable? likely not?
```

Comment:
I also thought about this ...
If we are 100% sure that e.g. all 400 errors are not recoverable, but 500 are, then we could set the `:recoverable`.
On the other hand the http can use any http endpoint, so i would stick with recoverable for all HTTP error codes.